packages/slice-machine/tests/src/modules/simulator.test.ts

```diff
@@ -119,7 +119,6 @@ describe("[Simulator module]", () => {
     it("should open setup modal if checkSimulatorSetupCreator.failure action", () => {
       const saga = testSaga(failCheckSetupSaga);
```

Comment:
honestly. i think expectSaga is more power full as it'll execute side-effect ike network calls